src/com/softwaret/kpdf/routing/routes/PublicationRouting.kt

```diff
@@ -22,19 +19,40 @@ fun Routing.publications(controller: PublicationsController) {
     @Location("/publications")
     data class PostPublicationLocation(val name: String, val pdfBase64: String)
 
+    @Location("/publications")
+    data class PutPublicationLocation(val publicationId: Int, val pdfBase64: String)
```

Comment:
Fajny pomysł z prefixem metody do klasy aka `Get`, `Put` itd. ale czy suffix `Location` to nie trochę za dużo w stylu raczej jak klasa zacznie się od nazwy metody to będzie wiadomo, że to lokacja. Jeżeli pójdziemy w tym kierunku to w innych routingach do zmiany.